sdk/python/kfmd/metadata.py

```diff
@@ -46,6 +46,10 @@ def __init__(self,
       labels {object} Optional key/value string pairs to label the workspace.
     """
     # TODO(zhenghuiwang): check each field's type and whether set.
```

Comment:
Can you remove the this TODO?

Same below.

Reply:
OK. Got it.